lesson16/json/sports.json

```
@@ -5,135 +5,135 @@
   "image": "https://i.postimg.cc/2Sbn5c2j/sports-baseball-man-asia.png",
   "article": [
     {
-      "id": "spo01",
+      "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
       "date": "2023-07-05",
       "title": "サッカー日本代表強さの秘密",
       "comment": [
         {
-          "id": "spo0101",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "一男",
           "icon": "https://i.postimg.cc/xdHXWfWs/icon-medical-man07.png",
-          "detail": "一男さんのコメント"
+          "detail": "Jリーグが地域に根付いたこと。優秀な指導者が増えたことが大きいのかな？底上げが出来ている。最近は外国にルーツがある選手も増えてフィジカルでも負けてない。今後にも期待したいね。"
         },
         {
-          "id": "spo0102",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "一花",
           "icon": "https://i.postimg.cc/7PQqnvRb/icon-business-woman09.png",
           "detail": "一花さんのコメント,"
         },
         {
-          "id": "spo0103",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "四郎",
           "icon": "https://i.postimg.cc/W4WsWQ49/icon-medical-man12.png",
           "detail": "四郎さんのコメント,"
         },
         {
-          "id": "spo0104",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "五木",
           "icon": "https://i.postimg.cc/G3DLcg2f/business-man1-1-smile.png",
           "detail": "五木さんのコメント,"
         },
         {
-          "id": "spo0105",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "六",
           "icon": "https://i.postimg.cc/KYjMkPMN/cat2-1-idea.png",
           "detail": "六さんのコメント"
         },
         {
-          "id": "spo0107",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "七",
           "icon": "https://i.postimg.cc/6pRSsZ3M/icon-business-woman09.png",
           "detail": "七さんのコメント"
         },
         {
-          "id": "spo0108",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "八",
           "icon": "https://i.postimg.cc/KYjMkPMN/cat2-1-idea.png",
           "detail": "八さんのコメント"
         },
         {
-          "id": "spo0109",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "九郎",
           "icon": "https://i.postimg.cc/VN2WzGVM/icon-medical-man12.png",
           "detail": "九郎さんのコメント"
         },
         {
-          "id": "spo01 10",
+          "id": "e7fe598f-292c-4ccd-9705-916a6ba69a12",
           "name": "十和",
           "icon": "https://i.postimg.cc/vZBrfwRf/icon-business-woman11.png",
           "detail": "十和さんのコメント"
         }
       ]
     },
     {
-      "id": "spo02",
+      "id": "772e5153-823c-4aea-8b69-c999d1f3fba9",
       "date": "2023-07-05",
       "title": "WBC日本5連覇達成",
       "comment": [
```

IMO
Since it's an array, I thought it would be nice to make it plural.
```suggestion
      "comments": [
```

---

Thank you sae-san :）
I fixed that.
It made my code easier to read.
51cbfc9cad94d0b14d17d51c874ee85def2371ce